src/helpers/markdown.js

```diff
@@ -91,5 +89,12 @@ export const getTableOfContent = (source) => {
   const tokens = marked.lexer(source)
   // remove the first 2 tokens to avoid meta information
   // and then return every tokens that is of header type
-  return tokens.slice(2).filter(token => token.type === "heading")
+  const headers = tokens.slice(2).filter((token) => token.type === "heading")
+  headers.forEach((header) => {
+    header.text = header.raw
+      .trim()
+      .replace(/^#+/g, "")// removes '#' that are at the start
+      .replace(/[`*_\s]/g, " ")// remove special char that are used in markdown
+  })
+  return tokens.slice(2).filter((token) => token.type === "heading")
```

Comment:
variable `headers` is manipulated but left unused. should be return headers?
```suggestion
  const headers = tokens.slice(2).filter((token) => token.type === "heading")
  return headers.map((header) => {
    header.text = header.raw
      .trim()
      .replace(/^#+/g, "")// removes '#' that are at the start
      .replace(/[`*_\s]/g, " ")// remove special char that are used in markdown
    return header
  })
```